package.json

```diff
@@ -23,7 +25,7 @@
     "del": "5.1.0",
     "eclint": "2.8.1",
     "got": "9.6.0",
-    "gulp": "4.0.2",
+    "gulp": "^4.0.2",
```

Comment:
Please revert that change, we use Renovate to manage and update our dependencies.

Reply:
Sorry for the updates, I’ll do it asap